### PR TITLE
ref(tests) Use datetime helpers in more places

### PR DESCRIPTION
**Pull request by markstory**
Use the datetime helpers in tests more extensively. This removes many magic 19s.

### CODE REVIEW

**Review by evanpurkhiser — APPROVED** [review]
This is great stuff

**Review by wedamija — APPROVED** [review]

File: src/sentry/testutils/helpers/datetime.py [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-from django.utils import timezone
-
-
 def iso_format(date):
     return date.isoformat()[:19]
 
 
 def before_now(**kwargs):
-    return timezone.now() - timedelta(**kwargs)
+    return datetime.utcnow() - timedelta(**kwargs)
```
Comment (wedamija) [inline src/sentry/testutils/helpers/datetime.py]: Is this more correct? I thought we generally used `timezone.now()` around the codebase

**Review by wedamija — APPROVED** [review]
Still lgtm

File: src/sentry/testutils/helpers/datetime.py (hunk 1 above)
Comment (markstory) [reply src/sentry/testutils/helpers/datetime.py]: For the most part we do. Where we don't is when we talk to snuba (which uses its own 'now') and in acceptance tests where timezone.now() is frozen to 2013.

**Pull request merged**